sandbox/crate/config/crate.yml

```diff
@@ -2,3 +2,4 @@
 cluster.name: dev
 discovery.zen.ping.multicast.enabled: false
 psql.enabled: true
+stats.enabled: true
```

Comment:
My consideration is that we'll have these scheduled jobs running by default that might interfere with debugging, but we can switch if off if this is the case.